src/tests/payment/test_views.py

```diff
@@ -0,0 +1,176 @@
+from unittest import mock
+from unittest.mock import MagicMock, patch
+
+import pytest
+import stripe
+from django.urls.base import reverse_lazy
+from rest_framework import status
+
+from payment.models import StripeProduct, Subscription
+
+
+@pytest.fixture
+def client(authenticated_client):
+    return authenticated_client
+
+
+@pytest.mark.django_db
+class TestSubscriptionViewSet:
+
+    create_url = reverse_lazy("v1:payment:subscription-list")
+    cancel_user_url = reverse_lazy("v1:payment:subscription-user-cancel")
+
+    def test_endpoint(self):
+        assert self.create_url == "/api/v1/payment/subscription/"
+
+    def test_endpoint2(self):
+        assert self.cancel_user_url == "/api/v1/payment/subscriptions/user/cancel/"
+
+    def retrieve_url(self, pk):
+        return reverse_lazy("v1:payment:subscription-detail", kwargs={"pk": pk})
+
+    @patch("payment.views.stripe.checkout.Session.create")
+    @patch("payment.views.StripeProduct.objects.get")
```

Comment:
Whenever we mock patch add an assert on `called`, `call_count` or `call_args_list`.
If a code change ever made the mock obsolete, the test would say it